### PR TITLE
Gh5888_1 Fixed values displayed on quantity field 

### PR DESCRIPTION
**Pull request by nelucristian**
https://github.com/metasfresh/metasfresh/issues/5888

### CODE REVIEW

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java [hunk 1]
```diff
@@ -59,6 +63,8 @@ private CreatePickingCandidatesCommand(
 		this.shipmentScheduleId = request.getShipmentScheduleId();
 		this.pickFrom = request.getPickFrom();
 		this.pickingSlotId = request.getPickingSlotId();
+		final BigDecimal qty = request.getQtyToPick().toBigDecimal();
+		this.quantity = Quantity.of(qty, getShipmentScheduleUOM());
```
Comment (teosarca) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java]: * **why changing the UOM of a quantity without any UOM conversion?**

* about calling `getShipmentScheduleUOM()` - avoid calling non static methods in constructor! see for example: https://stackoverflow.com/questions/18348797/why-is-it-considered-bad-practice-to-call-a-method-from-within-a-constructor 

* in case there is a need for that information in constructor than the uom which is computed by `getShipmentScheduleUOM()` shall be fetched in constructor and shall be captured as field.
So basically the `getShipmentScheduleUOM()` should be a just a simple getter.

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package de.metas.handlingunits.picking.candidate.commands;
 
+import org.apache.commons.lang.WordUtils;
```
Comment (teosarca) [inline de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java]: avoid unused imports

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java (hunk 1 above)
Comment (nelucristian) [reply de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java]: Thanks for review. I agree with creating the Quantity object in the perform() method, because of the call of getShipmentScheduleUOM().
But what does it mean this "**why changing the UOM of a quantity without any UOM conversion?**"  ? So shall i do a conversion or is it ok to pass the Quantity object from request.getQtyToPick() in the perform() method?

File: de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java (hunk 2 above)
Comment (nelucristian) [reply de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/candidate/commands/CreatePickingCandidatesCommand.java]: yes, sorry, it was by mistake

**Comment by nelucristian** [comment]
I made changes. Please review again
